pages/common/ts-node.md

```diff
@@ -0,0 +1,28 @@
+# ts-node
+
+> Runs TypeScript code directly, without any compiling.
+> More information at: https://www.npmjs.com/package/ts-node
+
+- Execute a TS File `foobar.ts` without compiling ('node' + 'tsc')
+
+`ts-node {{foobar.ts}}`
+
+- TS-Node Loads tsconfig.json by default. To prevent that
+
+`ts-node --skip-project {{foobar.ts}}`
+
+- Loads TS-Node Help
+
+`ts-node --help`
+
+- Eval TS Code directly on the terminal
+
+`ts-node -e 'console.log("Hello World")'`
+
+- Run code in Script Mode for `foobar.ts`
+
+`ts-node --script-mode {{foobar.ts}}`
+
+- Only transpile the TS File `foobar.ts` to JS
+
+`ts-node --transpile-only {{foobar.ts}}`
```

Comment:
```suggestion
- Transpile a TS file to JS without executing it:

`ts-node --transpile-only {{path/to/file.ts}}`
```

Reply:
That's a lotta suggestions but Alright lol